app/build.gradle.kts

```diff
@@ -110,7 +110,18 @@ composeCompiler {
 }
 
 dependencies {
-    implementation("androidx.appcompat:appcompat:1.6.1")
+    // Unfortunately, ui tooling, and the markdown thing, still brings in the other material2 dependencies
+    // This is the "official" composeBom, but it breaks the imageviewer until 1.7 is released. See:
+    // https://github.com/LemmyNet/jerboa/pull/1502#issuecomment-2137935525
+    // val composeBom = platform("androidx.compose:compose-bom:2024.05.00")
+
+    val composeBom = platform("dev.chrisbanes.compose:compose-bom:2024.05.00-alpha03")
+    implementation(composeBom)
+    implementation("androidx.activity:activity-compose")
```

Comment:
I'm assuming only the androidx. namespace things can use this composeBom. 

I tried to do it with all of them at first, but some still required explicit versioning.

Reply:
My understanding was that only the libraries listed where were covered
https://developer.android.com/develop/ui/compose/bom/bom-mapping

But that is not case. So I am not sure, you can probably look at the dependencies of the BOM to find out. Which libs are covered.